### PR TITLE
Reduced fontsizes of all headings smaller than h1.

### PR DESCRIPTION
**Pull request by Translucent504**
In response to https://github.com/Software-Development-Pakistan/Software-Development-Pakistan.github.io/issues/35

### CODE REVIEW

File: _sass/imports/_general.scss [hunk 1]
```diff
@@ -164,8 +164,8 @@ ol {
   padding: 0 0 0 1.33333em;
 }
 
-li > ul,
-li > ol {
+li>ul,
```
Comment (dani2819) [inline _sass/imports/_general.scss:167]: When you saved the file, your editor also formatted it in a different standards as we are using in other `.scss` files. If you have enabled `format on save`, please disable it before saving. and thanks for the PR.

File: _sass/imports/_general.scss [hunk 2]
```diff
@@ -469,4 +469,4 @@ img {
   blockquote {
     font-size: 1.55556em;
   }
-}
+}
```
Comment (dani2819) [inline _sass/imports/_general.scss:472]: No EOF?

File: _sass/imports/_general.scss (hunk 1 above)
Comment (Translucent504) [reply _sass/imports/_general.scss:167]: Yeah, that's my bad. I'll get rid of auto formatting.

File: _sass/imports/_general.scss (hunk 2 above)
Comment (Translucent504) [reply _sass/imports/_general.scss]: Fixed.

**Review by azmaktr — COMMENTED** [review]
LGTM

**Review by azmaktr — APPROVED** [review]

**Pull request merged**